int16 data_a
int16 data_b
int16 data_c

